File: answers/01A.cpp
// 1
#include <iostream>
using namespace std;

int main()
{
    int a, b, c;
    cin >> a >> b >> c;
    if (a > 0 && a < 180 && b > 0 && b < 180 && c > 0 && c < 180 && a + b + c == 180)
        cout << "Yes";
    else
        cout << "No";

    return 0;
}

File: answers/01A.py
# 1
a, b, c = map(int, input().split())
if (a and b and c) and (a + b + c == 180):
    print("Yes")
else:
    print("No")


# 2
angles = [int(i) for i in input().split()]
print("Yes" if all(angles) and sum(angles) == 180 else "No")

# arsham_mahdiun @Arshamm1997
